pkgs/tools/misc/memtest86+/default.nix

```
@@ -1,31 +1,35 @@
 { lib, stdenv, fetchgit }:
 
-stdenv.mkDerivation {
+stdenv.mkDerivation (finalAttrs: {
   pname = "memtest86+";
-  version = "5.01-coreboot-002";
+  version = "6.00-beta2";
 
   src = fetchgit {
-    url = "https://review.coreboot.org/memtest86plus.git";
-    rev = "v002";
-    sha256 = "0cwx20yja24bfknqh1rjb5rl2c0kwnppzsisg1dibbak0l8mxchk";
+    url = "https://github.com/memtest86plus/memtest86plus";
+    rev = "v${finalAttrs.version}";
```

```suggestion
    rev = "v${version}";
```
and add rec which is still faster.